src/pkg/packaging/windows/targetingpack/generatemsi.ps1

```diff
@@ -0,0 +1,162 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+param(
+    [Parameter(Mandatory=$true)][string]$TargetingPackPublishRoot,
+    [Parameter(Mandatory=$true)][string]$TargetingPackMSIOutput,
+    [Parameter(Mandatory=$true)][string]$WixRoot,
+    [Parameter(Mandatory=$true)][string]$ProductMoniker,
+    [Parameter(Mandatory=$true)][string]$TargetingPackMSIVersion,
+    [Parameter(Mandatory=$true)][string]$TargetingPackNugetVersion,
+    [Parameter(Mandatory=$true)][string]$Architecture,
+    [Parameter(Mandatory=$true)][string]$TargetArchitecture,
+    [Parameter(Mandatory=$true)][string]$WixObjRoot,
+    [Parameter(Mandatory=$true)][string]$TargetingPackUpgradeCode
+)
+
+$RepoRoot = Convert-Path "$PSScriptRoot\..\..\..\..\.."
+$CommonScript = "$RepoRoot\tools-local\scripts\common\_common.ps1"
+
+if(-Not (Test-Path "$CommonScript"))
+{
+    Exit -1
+} 
+. "$CommonScript"
+
+$PackagingRoot = Join-Path $RepoRoot "src\pkg\packaging"
+
+$InstallFileswsx = "$WixObjRoot\install-files.wxs"
+$InstallFilesWixobj = "$WixObjRoot\install-files.wixobj"
+
+function RunHeat
+{
+    $result = $true
+    pushd "$WixRoot"
+
+    Write-Host Running heat.. to $InstallFileswsx
+    write-host "Root $TargetingPackPublishRoot"
+
+    .\heat.exe dir `"$TargetingPackPublishRoot`" `
+    -nologo `
+    -template fragment `
+    -sreg -gg `
+    -var var.TargetingPackSrc `
+    -cg InstallFiles `
+    -srd `
+    -dr DOTNETHOME `
+    -out $InstallFileswsx | Out-Host
+
+    if($LastExitCode -ne 0)
+    {
+        $result = $false
+        Write-Host "Heat failed with exit code $LastExitCode."
+    }
+
+    popd
+    return $result
+}
+
+function RunCandle
+{
+    $result = $true
+    pushd "$WixRoot"
+
+    Write-Host Running candle..
+    $AuthWsxRoot =  Join-Path $PackagingRoot "windows\targetingpack"
```

Comment:
Inconsistency in casing of local variables.  PascalCased vs camelCased - $AuthWsxRoot vs $result